### ISSUE TITLE
Docs: Missing *Fork me on GitHub* image

### ISSUE THREAD

**Issue by piotr-cz** [thephpleague/glide#423]
[Docs](https://glide.thephpleague.com/) are missing *Fork me on GitHub* and show only alt text.

Requests to:

- img `src` attribute responds with *403 Forbidden* status (Bad signature)

- img `data-canonical-src` attribute responds with *403 Forbidden* status (AccessDenied)
